src/openrct2/peep/Staff.cpp

```diff
@@ -1449,17 +1445,15 @@ void Staff::UpdateHeadingToInspect()
         int32_t destX = NextLoc.x + 16 + DirectionOffsets[PeepDirection].x * 53;
```

Comment:
Should also be changed to use CoordsXY, but may be left for another PR (in which case I'd like you to create an issue for it.)